src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -0,0 +1,88 @@
+import { clone, reduce, keys } from 'underscore';
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import StateModel from './schedule_state';
+
+import FiltersApp from './filters_app';
+
+import { LayoutView, ScheduleTitleView, TableHeaderView } from 'js/views/patients/schedule/schedule_views';
+
+export default App.extend({
+  StateModel,
+  childApps: {
+    filters: {
+      AppClass: FiltersApp,
+      regionName: 'filters',
+      restartWithParent: false,
+      getOptions: ['currentClinician'],
+    },
+  },
+  stateEvents: {
+    'change:filters': 'restart',
+  },
+  onBeforeStart() {
+    if (this.isRestarting()) {
+      this.showScheduleTitle();
+      this.showTableHeaders();
```

Comment:
The title restarting makes sense I suppose because it changes based on the owner state.. _but_ that view itself has code to change based on the state, but we don't pass the state to that view...